docs/integrating-jupyter.md

```diff
@@ -0,0 +1,342 @@
+---
+title: Integrating Jupyter based Computation
+short_title: Integrating Jupyter
+description: blah
+thumbnail: thumbnails/integrating-jupyter.png
+---
+
+MyST allows you to connect a website directly to a Jupyter Kernel, enabling interactive computation on your page.
+
+This allows you to do some amazing things with your MyST website like:
+
+- Allow readers to recompute your notebooks for themselves, demonstrating the reproducibility of your work
+- Provide a new level of interactive content via `ipywidgets` and `ipywidgets` backed libraries such as `ipympl`, `ipyleaflet`, etc...
+- Connect to Binder based compute services
+- 🚧 Compute using the in-browser `pyodide` WASM kernel (backed by JupyterLite)
+
+```{attention}
+Integrated Jupyter Computation is hot off the press and currently limited to pages that represent complete Jupyter notebooks.
+We are in active development and this feature should be considered `beta` - please help us out [report any issues that you find](https://github.com/executablebooks/mystjs/issues).
+
+While being able to connect a jupyter based figure or output in any website page to a kernel is still work in progress - but expected very soon. The remainder of the docs below are forward looking, watch for the 🚧 icons on headings what are still work in progress.
+
+```
+
+## Quick setup options
+
+MyST uses `thebe` for Jupyter connectivity which can be enabled using defualt settings by adding a single key (`thebe:true`) to the project frontmatter in your `myst.yml` file.
+
+```{code} yaml
+:caption: an example myst.yml file with thebe enabled
+version: 1
+project:
+    title: Geocomputing
+    thebe: true
+site:
+    template: book-theme
+    title: My Computational Website
+```
+
+When the boolean form of the `thebe` key is used, MyST will try to determine where to connect to from existing `github` and `binder` keys you may have on your project and is intended to allow for easy setup of a few key use cases. To go beyond or override these, you can provide various options in the `thebe` field.
+
+### Case - `thebe:true` and no `github` or `binder` keys are present
+
+```yaml
+project:
+  thebe: true
+```
+
+When `thebe:true` and no `github` or `binder` keys are present MyST will try to connect to a server using the default (local settings). To make this work you'll need to [](#start-a-local-jupyter-server) with the correct defaults or [provide alternative direct connection options](#directly-connecting-to-a-jupyter-server).
+
+Note this is equivalent to:
+
+```yaml
+project:
+  thebe:
+    server: true
+```
+
+### Case - `thebe:true` and the `github` key is present
+
+```yaml
+project:
+  github: executablebooks/thebe-binder-base
+  thebe:
```

Comment:
Should this just be `thebe: true`? This config and the "equivalent" config below are identical.